### PR TITLE
Enable writeLaunchIdToFile

### PR DESCRIPTION
**Pull request by mikesalvia**

### CODE REVIEW

**Comment by mikesalvia** [comment]
@BorisOsipov Here is the other changes.

File: src/index.js [hunk 1]
```diff
@@ -19,6 +27,9 @@ class RpService {
 
     const {id} = await promise;
     process.env.RP_LAUNCH_ID = id;
+    if (this.options.writeLaunchIdToFile) {
```
Comment (BorisOsipov) [inline src/index.js:30]: would be better to make not just boolean option, but path to `RP_LAUNCH_ID`
 file

Comment (BorisOsipov) [reply src/index.js:30]: but it is not required

Comment (mikesalvia) [reply src/index.js]: Done! If `true` is supplied it will default to `RP_LAUNCH_ID`.

**Comment by mikesalvia** [comment]
@BorisOsipov Checking back in on this one... Any other edits requested?

**Comment by BorisOsipov** [comment]
released as 6.4.0

**Pull request merged**